phpBB/adm/style/acp_profile.html

```diff
@@ -17,7 +17,7 @@
 		</div>
 	<!-- ENDIF -->
 
-	<form id="add_profile_field" method="post" action="{U_ACTION}">
+	<form id="add_profile_field" method="post" action="{U_ACTION}"{S_FORM_ENCTYPE}>
```

Comment:
Again, no, this would not be compatible with current uses.
S_FORM_ENCTYPE content should be ' enctype="multipart/form-data"'. Otherwise, this would be incompatible with the current use pattern in ucp_profile_profile_info, that should be identical because it is configured the same way.